astrokat/test/test_offline_observe.py

```diff
@@ -95,11 +98,13 @@ def assert_started_target_scan(
                 duration, target_string
             )
         )
+        sim_coordinates_string = re.findall(CORELIB_RADEC_REGEX, sim_radec)[0]
+        corelib_coordinates_string = re.findall(CORELIB_RADEC_REGEX, corelib_radec)[0]
```

Comment:
Sorry, but this doesn't look useful at all.  The original string is being extracted using a regex.

```python
In [1]: import re

In [2]: CORELIB_RADEC_REGEX = r"\d\d\:\d\d\:\d\d\.\d+\s-?\d+:\d\d\:\d\d\.\d+"

In [3]: corelib_radec = "15:59:15.06 8:50:58.7"

In [4]: re.findall(CORELIB_RADEC_REGEX, corelib_radec)[0]
Out[4]: '15:59:15.06 8:50:58.7'
```

The idea of the regex is to use it when searching the logs.  Rather pass the regex pattern to `assert_started_target_scan` function, so that it can apply it.  The regex pattern I gave @mamkhari was much more specific.

Here's an example.  I've used regex for both sim and corelib coordinates to make the code more consistent.

```python
    def test_targets_sim(self):
...
        self.assert_started_target_scan(
            "target4_azel",
            duration=10.0,
            az=10.0,
            el=50.0,
            sim_radec_regex=r"16:00:05.19 8:50:57.6",
            corelib_radec_regex=r"15:59:[0-5]\d\.\d+ 8:50:[0-5]\d\.\d+,
            logs=result
        )

...

    def assert_started_target_scan(
        self, target_string, duration, az, el, sim_radec_regex, corelib_radec_regex, logs
    ):
        ...
        simulate_radec_message_regex = r"{}, tags=radec target, {}".format(
            target_string, sim_radec_regex
        )
        katcorelib_radec_message_regex = r"{}, tags=radec target, {}".format(
            target_string, corelib_radec_regex
        )
        simulated_messages_found = (
            simulate_message in logs
            and re.search(simulate_radec_message_regex, logs, re.MULTILINE) is not None
        )
        katcorelib_messages_found = (
            katcorelib_message in logs
            and re.search(katcorelib_radec_message_regex, logs, re.MULTILINE) is not None
        )
```